### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by sunivu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# SoftClassTeam4
+
+
+Test select reviewer
```
Comment (banseok1216) [inline README.md:4]: nice!

**Review by banseok1216 — COMMENTED** [review]
nice everything

File: README.md (hunk 1 above)
Comment (foobank) [inline README.md:4]: 뿌애앵

**Comment by banseok1216** [comment]
123

**Comment by foobank** [comment]
ㅇㅇㅇ

**Pull request merged**